src/Teams/Models/Team.cs

```diff
@@ -0,0 +1,28 @@
+﻿using System;
+using System.Collections.Generic;
+using System.ComponentModel.DataAnnotations;
+using System.ComponentModel.DataAnnotations.Schema;
+
+namespace Teams.Models
+{
+    [Table("team")]
+    public partial class Team
+    {
+        public Team()
+        {
+            TeamMembers = new HashSet<TeamMembers>();
```

Comment:
Why hashset? Can you guarantee that the TeamMembers class within Hashset will behave as a valid unique entries? How HashSet is working?